### ISSUE TITLE
Accessibility is unable to click within an emulator

### ISSUE THREAD

**Issue by KaitoKid** [Fate-Grand-Automata/FGA#1565]
### Preparation

- [X] I tested the [latest release](https://github.com/Fate-Grand-Automata/FGA/releases/tag/canary)
- [X] I looked at other issues (even the [closed ones](https://github.com/Fate-Grand-Automata/FGA/issues?q=is%3Aissue+is%3Aclosed))
- [X] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)

### FGO server

NA

### FGA build number

1951

### Describe the bug

I've got a bit of an interesting setup, so I'm looking for pointers rather than solutions as this probably is not applicable to everyone.

My setup:
- Android11 emulator in x86, with libndk_translation to run arm apks, running in a docker container courtesy of[redroid](https://github.com/remote-android/redroid-doc) 
- The recording is captured through quicktime as I'm using scrcpy to access the emulator running on a home server

Relevant information:
- Using root for screenshots
- The FGA is definitely able to identify elements on the screen as the support image maker worked perfectly, and the bounding boxes seem to be accurate as well
- Battery optimization is off for both FGA and FGO

Given the above, I believe the root cause to be the fault of 'Accessibility' failing to click, as the FGA seems to be expecting command cards.

Would appreciate if someone could validate this through reviewing the video.

### Video

https://www.dropbox.com/s/vyp20cio4wajsmj/FGA_Debug.mov?dl=0

### Device model

ReDroid

### Android version

11

### Screen size

1920x1080
